### DISCUSSION TITLE
Limitations of Zigbee connections of a device managed via Zigbe2Mqtt

### DISCUSSION THREAD

**Discussion by Spartacus68** [Koenkk/zigbee2mqtt#20632] [General]
Hello everyone

I have some older zigbee devices that now support of  20 connections max. (vendor told me). However, since my network is quite large, more than 20 devices connect dynamically to this device. This happens automatically and you can't prevent it. Is it possible to prohibit the connections that can be viewed via the frontend-map or to delete them individually so that they are not established automatically?

![image](https://github.com/Koenkk/zigbee2mqtt/assets/121975453/29c4b9ca-4602-45d7-878d-4de0e6947661)

**Comment by Koenkk** [comment]
Zigbee2MQTT cannot control how devices connect with each other

**Comment by Spartacus68** [comment]
ok, this means you cannot force a specific connection e.g. to the coordinator!

**Reply by Koenkk** [reply]
No, this is not possible.

**Comment by Spartacus68** [comment]
ok, I'm just thinking out loud...., but is there perhaps a way to configure the coordinator only to allow connections within a specific LQI range?
